modin/backends/pandas/query_compiler.py

```diff
@@ -545,6 +545,12 @@ def transpose(self, *args, **kwargs):
 
     # END String map partitions operations
 
+    def repeat(self, repeats):
+        new_modin_frame = self._modin_frame._apply_full_axis(
```

Comment:
We can make a more efficient version if `repeats` is an integer using `MapFunction`.

Reply:
yes, but because of `repeat` changes partitions sizes, we also should update indexes of `modin_frame`. Done it in the last commit